upgrade/1.4.2/README.md

```diff
@@ -174,6 +174,42 @@ This step will install the `smart-mon` rpm on storage nodes, and reconfigure the
    /usr/share/doc/csm/scripts/operations/ceph/enable-smart-mon-storage-nodes.sh
    ```
 
+## Configure NCN nodes without restart
+
+This step will create an imperative CFS session that can be used to configure booted NCN nodes with updated sysctl values.
+
+1. (`ncn-m001#`) Create a new CFS Configuration Entry for the release
```

Comment:
This role exists in `ncn-{master,worker,storage}.yml` and `ncn_nodes.yml`, when are those playbooks ran (do we need this step at all)?

Reply:
We need a named configuration within CFS that references this top level playbook. The most recent push calls this configuration 'ncn_nodes'. It can be anything, and its not particularly meaningful or relevant given this is run once (imperatively) and doesn't need to be referenced thereafter.